SparseGrids/gridtestExternalTests.cpp

```diff
@@ -215,101 +262,125 @@ TestResults ExternalTester::getError(const BaseFunction *f, TasGrid::TasmanianSp
             err += std::abs(y[j] - r[j]);
         };
         R.error = err;
-    }else if (type == type_internal_interpolation){
-        // load needed points
-        loadValues(f, grid);
-
-        std::vector<double> err(num_outputs, 0.0); // absolute error
-        std::vector<double> nrm(num_outputs, 0.0); // norm, needed to compute relative error
-
-        std::vector<double> test_x = genRandom(num_mc, num_dimensions);
-        std::vector<double> result_tasm(num_mc * num_outputs);
-        std::vector<double> result_true(num_mc * num_outputs);
+    }else if (type == type_internal_interpolation or type == type_internal_differentiation){
+        if (type == type_internal_differentiation and (!grid.isGlobal() or grid.isSequence())) {
```

Comment:
I don't think the if-statement is correct, are you missing `()` around `isGlobal() or isSequence()`

Reply:
Fixed.